index.html

```diff
@@ -216,6 +216,61 @@ <h3 class="poppins-font font-white">Skills</h3>
         </section>
       </div>
      </section>
+
+     <!--Section 4 | Get Started Section (Contact form) for mobile screens-->
+     <div class="section-4-mobile" id="section-4-mobile">
+      <img src="images/image-1-1.svg" alt="Geometric image in top corner" />
+
+      <div>
+        <h2 class="section-4-heading-mobile playfair-display-font font-color" id="section-4-heading-mobile">Get started</h2>
+
+        <p class="section-4-paragraph-mobile poppins-font font-color" id="section-4-paragraph-mobile">Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+        sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+      </div>
+
+      <form action="https://formspree.io/f/xwkyyayl" method="POST" name="contact-form" class="contact-form" id="contact-form">
+        <div class="user-name-section" id="user-name-section">
+          <label for="user-name" class="poppins-font">Your name</label>
+          <input type="text" class="poppins-font user-name" id="user-name" name="user-name" placeholder="Anastasiia" maxlength="30" required/>
+        </div>
+        
+        <div class="user-email-section" id="user-email-section">
+          <label for="user-email" class="poppins-font">Your email</label>
+          <input type="text" class="poppins-font user-email" id="user-email" name="user-email" placeholder="anastassia@gmail.com" required/>
+        </div>
+
+        <textarea class="poppins-font" id="user-comments" name="user-comments" placeholder="Enter your message here..." maxlength="500"></textarea>

```

Comment:
- [ ] Please make sure to add the `required` property to your textarea, so your user cannot submit a form without a text.